src/Winton.Extensions.Configuration.Consul/Winton.Extensions.Configuration.Consul.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageTags>.NET;Core;dotnetcore;ASP.NET;aspnetcore;configuration;consul;winton;wintoncode</PackageTags>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/wintoncode/Winton.Extensions.Configuration.Consul</RepositoryUrl>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
As I mentioned in the issue, let's try reverting this and adding the following back to the bottom of the csproj to see if we can get it to run under net461 again and also pass your vulnerability scan.

```xml
<ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
  <Reference Include="System.Net.Http" />
  <Reference Include="System.Net.Http.WebRequest" />
</ItemGroup>
```

Reply:
done

Reply:
Does it run under `net461` now? And does your vulnerability scanner still pass?

Reply:
It does run under net461. I'm going to do an initial assessment regarding the vulnerability scanner part

Reply:
I'm unable to run the scan locally but when I compare the dependencies of the older Consul package (v. 0.7.2.6) and the new one (1.6.10.4), I see that the new version got rid of many System.Net dependencies (among which the offending System.Net.Http and System.Net.Http.WinHandler). So I'm confident that upgrading the consul package will do the trick for us.

Reply:
Nice one that sounds good. I think this is a good change anyway, so it should be released and if for some reason there are still issues with the vuln scanner afterwards we can readdress those. 

I'm not actually a project owner anymore since I no longer work at Winton, so I'll see if someone that does can approve and merge. Thanks for fixing.

Reply:
Thank you for your time!